geo.txt

```diff
@@ -56,3 +56,1703 @@ Phys 91(11):957-962
+complexes: a comparative study of a weak C-H⋅⋅⋅N hydrogen bond and an S⋅⋅⋅N bond. Phys Chem Chem Phys
+13(31):14235-14242
+# Date first added: 2023-07-11 (by Sichao Xiong)
+
+H...N    2.3980(7)
+NH3 HCCH 4.0594(6)
+α        23.29(2)
+
+# α: Angular oscillation of ammonia subunit. 
+
+NAME = C2N2 2011
+# Point Group: Dooh 
+# Shape: Linear
+# Bond lengths: re (angstroms)
+# Source paper: Maki AG (2011) High-resolution infrared spectrum of cyanogen. J Mol Spectrosc 269(2):166-174
+# Date first added: 2023-07-11 (by Sichao Xiong)
+
+C   C 1.38109(60)
+C ≡ N 1.15976(40)
+
+NAME = C2N2S 2013
+# Point Group: C2v
+# Shape: Bent
+# Bond lengths: r0     (angstroms)
+# Bond angles:  theta0 (degrees)
+# Source paper: Kisiel Z, Winnewisser M, Winnewisser BP, DeLucia FC, Tokaryk DW, Billinghurst BE (2013) Far-infrared
+spectrum of S(CN)2 measured with synchrotron radiation: global analysis of the available high-resolution
+spectroscopic data. J Phys Chem A 117(50):13815-13824
+# Date first added: 2023-07-11 (by Sichao Xiong)
+
+S   C     1.698(6)
+C ≡ N     1.160(9) 
+C   S C   98.6(5)
+S   C ≡ N 175.4(11)
+
+NAME = C2OS3 2010
+# Point Group: Cs
+# Shape: Linear --- Linear
+# Bond lengths: r0     (angstroms)
+# Bond angles:  theta0 (degrees)
+# Source paper: Norooz Oliaee J, Mivehvar F, Dehghany M, Moazzen-Ahmadi N (2010) Geometric isomerism in the OCS-CS2
+complex: Observation of a cross-shaped isomer. J Phys Chem A 114(27):7311-7314
+# Date first added: 2023-07-11 (by Sichao Xiong)
+
+S2C2O  SC1S        3.5553(8)
+C(1)...C(2) = S(2) 104.82(22)
+
+NAME = C2O3
+# Point Group: C2v
+# Shape: Linear --- Linear
+# Bond lengths: r0     (angstroms)
+# Source paper: Sheybani-Deloui S, Barclay AJ, Michaelian KH, McKellar ARW, Moazzen-Ahmadi N (2015) Spectroscopic
+observation of the O-bonded T-shaped isomer of the CO-CO2 dimer and two of its intermolecular frequencies. J
+Chem Phys 143(12):121101/1-121101/4
+[http://dx.doi.org/10.1063/1.4932043]
+# Date first added: 2023-07-11 (by Sichao Xiong)
+
+OCO OC 3.583
+
+NAME = C2O3S 
+# Point Group: Cs
+# Shape: Linear --- Linear
+# Bond lengths: r0     (angstroms)
+# Source paper: a. Dehghany M, Norooz Oliaee J, Afshari M, Moazzen-Ahmadi N, McKellar ARW (2009) Infrared spectra of
+the OCS-CO2 complex: Observation of two distinct slipped near-parallel isomers. J Chem Phys 130(22):224310/1-224310/5 doi: 10.1063/1.3152743 b. Sedo G, van Wijngaarden J (2009) Fourier transform microwave spectra of a “new” isomer of OCS-CO2. J
+Chem Phys 131(4) 044303/1-044303/5
+doi: 10.1063/1.3186756
+# Date first added: 2023-07-11 (by Sichao Xiong)
+
+# O-interior
+
+OCO SCO 3.9862(10)
+θ1      118.0 (44) 
+θ2      135.8 (24)
+
+# MW
+
+OCO SCO 3.9816(16)
+θ1      108.8(46)
+θ2      140.7(46)
+
+# S-interior
+
+OCO SCO 3.5545 (10)
+θ1      69.7 (34)
+θ2      78.7 (25)
+
+# θ1: Angle between Rcm and the O=C=O axis (measured from the inner O end)
+# θ2: Angle between Rcm and the O=C=S axis (measured from the S end).
```

Comment:
Ideally our data format would have some mechanism to portray this information without having to write it.